api/README.md

```diff
@@ -13,7 +13,7 @@ To be updated
 To start, first create a python virtual environment:
 
 ```bash
-virtualenv env --python=python3
+virtualenv env --python=python3 --copies
```

Comment:
From the virtualenv help cmd:

> Try to use copies rather than symlinks, even when symlinks are the default for the platform.

For mac, the default is symlinks when you try to invoke a virtualenv. Instead of having a local python interpreter, they would use a global one (our operating system's python). So what `--copies` do here is basically it would copy that global python interpreter into our local env